### PR TITLE
Reco new command exclusion

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 1]
```diff
@@ -1251,6 +1272,10 @@ def main() -> None:
                                                        demisto.args()["domain"],
                                                        int(demisto.args()["last_interaction_time_in_days"]))
             return_results(result)
+        elif command == "reco-add-exclusion-filter":
+            result = add_exclusion_filter(reco_client, demisto.args()["key_to_add"],
+                                          demisto.args()["values_to_add"].split(','))
```
Comment (mmhw) [inline Packs/Reco/Integrations/Reco/Reco.py:1277]: You can use `argToList`
```suggestion
                                          argToList(demisto.args()["values_to_add"]))
```

File: Packs/Reco/Integrations/Reco/Reco.yml [hunk 2]
```diff
@@ -65,13 +65,23 @@ description: Reco is a Saas data security solution that protects your data from
...
   runonce: false
   script: "-"
   subtype: python3
   type: python
   commands:
+  - arguments:
+    - description: Values to add to the exclusion filter (split by ',')
+      name: values_to_add
+      required: true
+      type: List
```
Comment (mmhw) [inline Packs/Reco/Integrations/Reco/Reco.yml:79]:
```suggestion
      type: List
      isArray: true
```

**Review by mmhw — COMMENTED** [review]
Hi @GalNakash-RecoLabs,
Thank you for your contribution!
Good work :)

Please note my comments below.

Please feel free to reach out to me with any questions - I'm available here or on slack :)
Thanks again

**Comment by GalNakash-RecoLabs** [comment]
> Please bump the pack version in the `pack_metadata.json` file and add notes describing your changes to a new Release Notes file. To do that, please run the `demisto-sdk update-release-notes -i Packs/Reco` command - for more information please see [here](https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes).

Done

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 3]
```diff
@@ -1015,7 +1030,13 @@ def get_sensitive_assets_by_name(reco_client: RecoClient, asset_name: str, regex
     return assets_to_command_result(assets)
 
 
-def assets_to_command_result(assets: List[Dict[str, Any]]) -> CommandResults:
+def add_exclusion_filter(reco_client: RecoClient, key_to_add: str, values: list[str]) -> CommandResults:
+    """Add exclusion filter to Reco."""
+    response = reco_client.add_exclusion_filter(key_to_add, values)
+    return CommandResults(raw_response=response)
```
[the remark this answers is not in the document]
Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/Reco.py]: No, good catch. Added

**Comment by mmhw** [comment]
Hi @GalNakash-RecoLabs,
Is the PR ready for you?

**Comment by GalNakash-RecoLabs** [comment]
> Hi @GalNakash-RecoLabs, Is the PR ready for you?

Yes

**Review by mmhw — APPROVED** [review]
Good work!

**Pull request merged**